collectors/ebpf.plugin/ebpf_apps.h

```diff
@@ -11,7 +11,7 @@
 #include "libnetdata/ebpf/ebpf.h"
 
 #define NETDATA_APPS_FAMILY "apps"
-#define NETDATA_APPS_FILE_GROUP "file (eBPF)"
+#define NETDATA_APPS_FILE_GROUP "file_access"
```

Comment:
Why did you change that? You use `(eBPF)` everywhere.

Reply:
When we were using `file (eBPF)` the new links created to move between global charts and apps charts were not working, so we are renaming it to have conditions to create the needed links.